classes/class4/homework/server.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var exphbs = require('express-handlebars');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+
+var app = express();
+
+/*MongoLab Environment vars*/
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+mongoose.connect(mongoURI);
+var PORT = process.env.PORT || 3000;
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+/*Routes with cats.js*/
+app.use('/cats', require('./routes/cats.js'));
```

Comment:
Avoid in-line `require`'s. 
Just put `var cat_routes = require('./routes/cats.js')` above, and use
`app.use('/cats', cat_routes)` down below.